src/Tools/Test Framework/Test Runner/src/DataDrivenTest/DataOutputs/TestOutput.Table.al

```diff
@@ -0,0 +1,82 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+namespace System.TestTools.TestRunner;
+
+table 130453 "Test Output"
+{
+    DataClassification = CustomerContent;
+    TableType = Temporary;
+
+    fields
+    {
+        field(1; "Test Suite"; Code[10])
+        {
+            TableRelation = "AL Test Suite".Name;
+            Caption = 'Test Suite';
+            Tooltip = 'Specifies the test suite code.';
```

Comment:
Name or code? According to the table relation, it's a name.
```suggestion
            Tooltip = 'Specifies the test suite name.';
```

Reply:
It is wrongly named in the existing tool, lets keep it as it is.